src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
Yep, a separate TUI wrapper script is a good idea! I was already thinking on how to make that cleaner.
As for -l VS --list, it doesn't matter for me. I just couldn't figure out whether getopts works with long options.

If you have more feedback/suggestions/idea, let me know! Thanks!